.github/workflows/build.yml

```diff
@@ -254,14 +256,46 @@ jobs:
           echo "${{ steps.changelog.outputs.changelog }}" > ./CHANGELOG.md
           sed -i "1s/.*/## Changes since ${{ steps.last-release.outputs.release }}/" ./CHANGELOG.md
 
-      - name: Create the release
-        uses: actions/create-release@v1
+      # automatic helm repo management
+      # (from https://github.com/liqotech/liqo/blob/master/.github/workflows/integration.yml)
+      - name: Install Helm
+        uses: azure/setup-helm@v1
         with:
-          tag_name: ${{ needs.configure.outputs.version }}
-          release_name: Release ${{ needs.configure.outputs.version }}
-          body_path: ./CHANGELOG.md
-        env:
-          GITHUB_TOKEN: ${{ secrets.CI_TOKEN }}
+          version: v3.8.1
+
+      - name: Download chart releaser
+        run: |
+          curl -sSL "https://github.com/helm/chart-releaser/releases/download/v1.4.0/chart-releaser_1.4.0_linux_amd64.tar.gz" | tar -xz
+
+      - name: Package helm chart
+        run: |
+          # the output should be in the .cr-release-packages since cr index expects to find it there to create the helm index
+          helm package deploy/crownlabs --version "${{ needs.configure.outputs.version }}" --app-version "${{ needs.configure.outputs.version }}" --destination .cr-release-packages
+
+      - uses: ncipollo/release-action@v1
+        with:
+          artifacts: ".cr-release-packages/*"
+          generateReleaseNotes: true
```

Comment:
With this set to true, it should no longer be necessary the manual generation of the changelog.